Packages/Status/Sources/Status/Row/StatusRowViewModel.swift

```diff
@@ -37,6 +37,21 @@ public class StatusRowViewModel: ObservableObject {
   @Published var localStatusId: String?
   @Published var localStatus: Status?
 
+  // the next three values will come from user settings
+  let collapseLongPosts = true
+  let collapseThresholdLength : Int = 300
+  let _collapsedLines: Int = 5  
```

Comment:
I think we coud just have a configurable line limit with a default to 7 or 8 and that could be it. I don't think we need characters count. Lines limit scale well with the device size.

Reply:
The character count is to determine if the collapse should be shown at all.  How would I know that a text exceeds the line count for the collapsed view and the button needs to be shown?

Reply:
🤦🏻 Oh right. Yes I don't see a way to know if we're over the line limit or not without it costing too much performance.